### ISSUE TITLE
[a11y]: Element "div" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'

### ISSUE THREAD

**Issue by xubinnb** [carbon-design-system/carbon#15550]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

v7.59.16

### React version

v17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

===== ENVIRONMENT =====
Browser: Version 105.0.5195.125 (Official Build) (arm64)
Automatically update Chrome for all users Learn more
Test Tool Type: IBM Accessbility Checker

===== DEFECT DESCRIPTION ===== 
Problem 1: Element "div" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'

=====STEPS TO REPRODUCE =====
Go to https://dev.console.test.cloud.ibm.com/projects
Click 'Create'
Input the name and description to create a new project
Go to 'Management' tab
Go to 'Environment' tab
Click 'Add'

Use IBM Equal Access Accessibility Checker to scan

===== EXPECTED RESULT =====
[IBM 1.3.1 Info and Relationships](https://www.ibm.com/able/requirements/requirements/#1_3_1)
Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.
[IBM 4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
For all user interface components (including but not limited to: form elements, links, and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.

===== SCREEN CAPTURE =====
<img width="1275" alt="image" src="https://media.github.ibm.com/user/42629/files/4da97a05-ae8f-43c0-b47d-1ff6dc29f0e9">

### Reproduction/example

NA

### Steps to reproduce

=====STEPS TO REPRODUCE =====
Go to https://dev.console.test.cloud.ibm.com/projects
Click 'Create'
Input the name and description to create a new project
Go to 'Management' tab
Go to 'Environment' tab
Click 'Add'

Use IBM Equal Access Accessibility Checker to scan

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tw15egan** [comment]
Can you please provide a simplified reproduction of the a11y violation via [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite) so we can take a deeper look?

**Comment by flannanl** [comment]
This should show the issue:

```
import { Button, Toggle, Tooltip } from '@carbon/react';
import React from 'react';

export default function App() {
  return (
    <div>
      <Toggle
        size="sm"
        id="toggle-1"
        labelA={
          <Tooltip
            triggerText="Yes"
            triggerId="trigger_yes"
            tooltipId="tooltip_yes"
          >
            <p>Some text</p>
          </Tooltip>
        }
        labelB={
          <Tooltip
            triggerText="No"
            triggerId="trigger_no"
            tooltipId="tooltip_no"
          >
            <p>Some text</p>
          </Tooltip>
        }
      />
    </div>
  );
}
```

**Comment by tay1orjones** [comment]
About a year ago some a11y updates were made to Toggle but they were never backported to v10. Could be that they just need backported to v10 (`carbon-components-react@7.x`)? Need to take a deeper look, but these are what I recall:

* https://github.com/carbon-design-system/carbon/pull/12974
* https://github.com/carbon-design-system/carbon/pull/13057
* https://github.com/carbon-design-system/carbon/pull/13196

**Comment by laurenmrice** [comment]
We will close this issue in favor of the older duplicate issue #14921.
